crates/router/src/core/payments/operations/payment_capture.rs

```diff
@@ -135,6 +138,17 @@ impl<F: Send + Clone> GetTracker<F, payments::PaymentData<F>, api::PaymentsCaptu
             .await
             .transpose()?;
 
+        let capture = match payment_attempt.capture_method {
+            Some(enums::CaptureMethod::ManualMultiple) => {
+                let new_capture = Self::create_capture(state, &payment_attempt, storage_scheme)
+                    .await
+                    .change_context(errors::ApiErrorResponse::InternalServerError)
+                    .attach_printable("Failed to create capture in DB")?;
+                Some(new_capture)
+            }
+            _ => None,
```

Comment:
don't use underscore, use if else for these kind of case(Only one variant needs to be handled)